privaterelay/settings.py

```diff
@@ -170,12 +172,7 @@
     ]
 
 
-def download_xpis(headers, path, url):
-    if path.endswith('.xpi'):
-        headers['Content-Disposition'] = 'attachment'
-
-WHITENOISE_ADD_HEADERS_FUNCTION = download_xpis
-
```

Comment:
I'm guessing this was for [signing Firefox add-ons](https://wiki.mozilla.org/Add-ons/Extension_Signing) which is now in separate repo?

Reply:
Well, it was when we though we would self-distribute the `.xpi` file thru the website. But we're linking over to AMO now.